README.md

```diff
@@ -92,30 +111,65 @@ echo "AVS signing address: $AVS_SIGNING_ADDRESS"
 # AVS_SIGNING_ADDRESS="0x..." # Address of the key that will sign for the AVS
 
 # Register the operator using the operator key and AVS signing address
-docker run --rm --network host --env-file .env \
+docker run --rm --network host \
+   -e DEPLOY_ENV=${DEPLOY_ENV} \
+   -e LOCAL_ETHEREUM_RPC_URL=${LOCAL_ETHEREUM_RPC_URL} \
+   -e TESTNET_RPC_URL=${TESTNET_RPC_URL} \
+   -e LST_CONTRACT_ADDRESS=${LST_CONTRACT_ADDRESS} \
+   -e LST_STRATEGY_ADDRESS=${LST_STRATEGY_ADDRESS} \
    -e WAVS_SERVICE_MANAGER_ADDRESS=${WAVS_SERVICE_MANAGER_ADDRESS} \
-   wavs-middleware register "$OPERATOR_KEY" "$AVS_SIGNING_ADDRESS" "1000000000000000"
+   -e OPERATOR_KEY=${OPERATOR_KEY} \
+   -e WAVS_SIGNING_KEY=${AVS_SIGNING_ADDRESS} \
+   -e WAVS_DELEGATE_AMOUNT=${WAVS_DELEGATE_AMOUNT} \
+   wavs-middleware register
 ```
 
 List Operators:
 
 ```bash
 # View stake registry status, including registered operators and their weights
-docker run --rm --network host  --env-file .env \
+docker run --rm --network host \
+   -e DEPLOY_ENV=${DEPLOY_ENV} \
+   -e LOCAL_ETHEREUM_RPC_URL=${LOCAL_ETHEREUM_RPC_URL} \
+   -e TESTNET_RPC_URL=${TESTNET_RPC_URL} \
    -e WAVS_SERVICE_MANAGER_ADDRESS=${WAVS_SERVICE_MANAGER_ADDRESS} \
    wavs-middleware list_operators
 ```
 
+Update Quorum:
+
+```bash
+export QUORUM_NUMERATOR=3
+export QUORUM_DENOMINATOR=5
+
+docker run --rm --network host -v ./.nodes:/root/.nodes \
+   -e DEPLOY_ENV=${DEPLOY_ENV} \
+   -e LOCAL_ETHEREUM_RPC_URL=${LOCAL_ETHEREUM_RPC_URL} \
+   -e TESTNET_RPC_URL=${TESTNET_RPC_URL} \
+   -e QUORUM_NUMERATOR=${QUORUM_NUMERATOR} \
+   -e QUORUM_DENOMINATOR=${QUORUM_DENOMINATOR} \
+   -e WAVS_SERVICE_MANAGER_ADDRESS=${WAVS_SERVICE_MANAGER_ADDRESS} \
+   wavs-middleware update_quorum
```

Comment:
So, with the parse_args function, you could also call this without the exports above and like:

`docker run ... wavs-middleware update_quorum QUORUM_NUMERATOR=3 QUORUM_DENOMINATOR=5` right?

Can you update the readme to show that usage here, and some other ones that are more user-defined rather than fixed configuration, eg `WAVS_DELEGATE_AMOUNT=1000000000000000` above (but WAVS_SERVICE_MANAGER makes perfect sense as env)

Reply:
fixed